src/python/grpcio/grpc/__init__.py

```diff
@@ -1967,6 +1967,59 @@ class Compression(enum.IntEnum):
     Gzip = _compression.Gzip
 
 
+def _uninstalled_protos(*args, **kwargs):
+    raise NotImplementedError(
+        "Install the grpcio-tools package to use the protos function.")
+
+
+def _uninstalled_services(*args, **kwargs):
+    raise NotImplementedError(
+        "Install the grpcio-tools package to use the services function.")
```

Comment:
Do we want to "brand" this feature as dynamic stubs here?

Reply:
No. I don't think that's a good idea. The branding is really more for internal buy-in than anything else. I don't want the user to have to know what a "stub" is or that there's a distinction between "static" and "dynamic". All I want them to have to know is "This is the function I call to get access to my protos and services".